TODO.md

```diff
@@ -0,0 +1,24 @@
+# TODO
+
+* volatile images
+
+* work on device nodes
+
+* allow passing env vars
+
+* mkosi --all (for building everything in mkosi.files/)
+
+* --architecture= is chaos: we need to define a clear vocabulary of
+  architectures that can deal with the different names of
+  architectures in debian, fedora and uname.
+
+* --boot-protocols=/BootProtocols= should be stricter and refuse
```

Comment:
isn't it doing that already? Last time I tried it was.

Reply:
ah, true, i was confused that it was not implemented as Enum (though probably should be)